README.md

```diff
@@ -8,3 +8,19 @@ COSE is scary. COSE has a very wide feature spectrum. It is quite complex.
 Complexity multiplied with a vast range of features amplify the trepidation.
 Hence when a software developer wants to pick a COSE Library for their project,
 one does not know, what one gets ?
+
+# Proposal
+* Have a clear view of tables that express the support of features by a certain COSE library, so that a user that needs to decide whether to use a certain crypto primitive for their software project can clearly identify the level of support for that feature in the open-source software ecosystem.
```

Comment:
```suggestion
Have a clear view of tables that express the support of features by a certain COSE library, so that a user that needs to decide whether to use a certain crypto primitive for their software project can clearly identify the level of support for that feature in the open-source software ecosystem.
```